src/main.rs

```diff
@@ -43,9 +57,16 @@ struct Opts {
     #[clap(long)]
     profile_out: Option<PathBuf>,
 
-    #[clap(long)]
     /// How many samples per seconds. Defaults to 500_000 (every 5us).
+    #[clap(long)]
     profile_frequency: Option<u32>,
+
+    #[clap(short = 'l', long, value_enum, default_value = "json")]
+    input_flavor: InputFlavor,
+
+    /// Log buffer size. Specify either a number or "unbounded" to disable the limit.
+    #[clap(short = 'b', long, default_value = "1000")]
+    log_buffer: LogMaxSize,
```

Comment:
We could probably drop the max altogether, and emit a warning if the log would've beeb truncated in production? Not sure when log truncation is ever desirable here.